test/canary/scripts/push_stats_to_cloudwatch.py

```diff
@@ -0,0 +1,94 @@
+import boto3
+from datetime import datetime
+import xml.etree.ElementTree as ET
+import os
+
+
+xml_path = "../integration_tests.xml"
+
+def readXML_and_publish_metrics_to_cw():
+    if os.path.isfile(xml_path):
+        tree = ET.parse(xml_path)
+        testsuite = tree.find("testsuite")
+        failures = testsuite.attrib["failures"]
+        tests = testsuite.attrib["tests"]
+        successes = int(tests) - int(failures)
+    else:
+        print("f{xml_path} does not exists.")
+        print(os.getcwd())
+        failures = 0
+        successes = 0
+        tests = 1
```

Comment:
Why is this 1? Shouldn't it be equal to the amount of tests in the canary suite?

Reply:
does it matter 0/1 is still 0?
yes it does give wrong sense of how many tests ran but making it equal to num of tests will not be maintainable

Reply:
yea it was there for making 0/1=0 so it still sends to cloudwatch if the log was not generated (no tests were run)